accord-core/src/main/java/accord/impl/InMemoryCommandStore.java

```diff
@@ -99,6 +100,7 @@ public abstract class InMemoryCommandStore extends CommandStore
 
     private final TreeMap<TxnId, RangeCommand> rangeCommands = new TreeMap<>();
     private final TreeMap<TxnId, Ranges> historicalRangeCommands = new TreeMap<>();
+    private final TreeSet<TxnId> historicalTxnIds = new TreeSet<>();
```

Comment:
switch on using the `Invariants.DEBUG` flag?

Reply:
I don't actually plan to merge this, its just something I needed for trying to cover more states... I might be able to push this to the delayed store (which is test only)

Reply:
removed.